README.md

```diff
@@ -129,14 +129,44 @@ GCC 4.9 / Clang 3.2 or later is recommended because OpenSpades relies on C++11 f
 9. Download the [Non-free pak](https://github.com/yvt/openspades-paks/releases/download/r33/OpenSpadesDevPackage-r33.zip), extract it, and copy `Nonfree/pak000-Nonfree.pak` to the `Resources` folder inside your build output directory, which is probably `E:/Projects/openspades/openspades.msvc/bin/BUILD_TYPE/Resources`. You can also copy the paks contained in `Official Mods/` folder of OpenSpades 0.0.12b to add more fonts and improve localization support of your build.
 
 ### On macOS (with Ninja)
-1. Get the latest version of Xcode and the OpenSpades source code. Install the following prerequisites:
-    - Xcode's command-line tools. Install them by `xcode-select --install` and `sudo xcodebuild -license`.
-    - CMake.
-    - Install `pkg-config` by one of the following methods:
-      - Homebrew: `brew install pkg-config`
-      - Nix: add `$(nix-build '<nixpkgs>' -A pkg-config-unwrapped --no-out-link)/bin` to `PATH`.
-2. `vcpkg/bootstrap-vcpkg.sh`
-3. `vcpkg/vcpkg install @vcpkg_x86_64-darwin.txt`
+
+#### Requirements
+- Xcode Command Line Tools
+- CMake
+- pkg-config
+- gcc 6 or newer (not clang!) — macOS 10.14 (Mojave) or earlier only
```

Comment:
why is that ? (not clang)

Reply:
nevermind saw your ticket.

Reply:
vcpkg relies on some specific functionality (Filesystem types?) that wasn't available until 10.15

running `vcpkg/bootstrap.sh` gives you this output:
```
CMake Error at cmake/utilities.cmake:30 (message):
  Building the vcpkg tool requires support for the C++ Filesystem TS.

  macOS versions below 10.15 do not have support for it with Apple Clang.

  Please install gcc6 or newer from homebrew (brew install gcc).

  If you would like to try anyway, pass --allowAppleClang to bootstrap.sh.
```

and then if you persist with --allowAppleClang you get a massive amount of errors:
```
[1/86] Building CXX object CMakeFiles/vcpkg.dir/src/vcpkg.cpp.o
FAILED: CMakeFiles/vcpkg.dir/src/vcpkg.cpp.o 
/Library/Developer/CommandLineTools/usr/bin/clang++ -DVCPKG_USE_STD_FILESYSTEM=1 -I../include -O3 -DNDEBUG -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX10.14.sdk -std=c++17 -MD -MT CMakeFiles/vcpkg.dir/src/vcpkg.cpp.o -MF CMakeFiles/vcpkg.dir/src/vcpkg.cpp.o.d -o CMakeFiles/vcpkg.dir/src/vcpkg.cpp.o -c ../src/vcpkg.cpp
In file included from ../src/vcpkg.cpp:4:
../include/vcpkg/base/files.h:46:5: error: 'path' is unavailable: introduced in macOS 10.15
    path u8path(vcpkg::StringView s);
    ^
/Library/Developer/CommandLineTools/usr/bin/../include/c++/v1/filesystem:739:24: note: 'path' has been explicitly marked unavailable here
class _LIBCPP_TYPE_VIS path {
                       ^

etc ad nauseum
```

Reply:
makes sense ! did not know, I never use vcpkg on *nix systems.

Reply:
yeah this was a first for me too!